core/src/main/java/org/openstack4j/model/network/ext/LoadBalancerV2.java

```diff
@@ -34,14 +34,14 @@
     String getDescription();
 
     /**
-     * @return The VIP subnet id of the loadbalancer.
+     * @return The vip subnet id of the loadbalancer.
      */
-    String getVIPSubnetId();
+    String getVipSubnetId();
 
     /**
-     * @return The VIP address of the loadbalancer.
+     * @return The vip address of the loadbalancer.
      */
-    String getVIPAddress();
+    String getVipAddress();
 
```

Comment:
The documentation does not show vip_port_id (https://wiki.openstack.org/wiki/Neutron/LBaaS/API_2.0#Retrieve_a_specific_Load_Balancer) but i'm seeing it when I test. json above